src/pybind11-qt/pybind11_qt_basic.cpp

```diff
@@ -68,8 +72,7 @@ namespace pybind11::detail {
                                       handle /* parent */)
     {
         static_assert(sizeof(QChar) == 2);
-        return PyUnicode_FromKindAndData(PyUnicode_2BYTE_KIND, src.constData(),
-                                         src.length());
+        return PyUnicode_FromString(src.toStdString().c_str());
```

Comment:
Any reason to change this? AFAIK, Unicode string in Python can be stored as UTF-16 so this adds an extra conversion from UTF-16 to UTF-8.

Reply:
Emoji characters can't be directly converted back to QStrings in the previous function. The current workaround is to convert to UTF-16. This allows us to directly use QStrings from C++ -> Python -> C++ without having to run character conversions in Python.